src/mlpack/methods/kmeans/hamerly_kmeans_impl.hpp

```diff
@@ -104,7 +107,7 @@ double HamerlyKMeans<DistanceType, MatType>::Iterate(const arma::mat& centroids,
 
       const double dist = distance.Evaluate(dataset.col(i), centroids.col(c));
 
-      // Is this a better cluster?  At this point, upperBounds[i] = d(i, c(i)).
+      // Is this a better cluster?  At this point, upperBounds[i] = d(i, c(i))
```

Comment:
```suggestion
      // Is this a better cluster?  At this point, upperBounds[i] = d(i, c(i)).
```
Since we aren't adding an extra indentation, the period still fits. :+1: